data.sql

```diff
@@ -7,3 +7,18 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', 'January 07, 2021', 1, '0', 15.04);
 
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', 'May 12, 2017', 5, '1', 11);
+
+/* Added for Project2 requierements*/
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Charmander', 'February 08, 2020', 0, '0', -11);
```

Comment:
Great job @Oklukeok 🤝

- [ ] [Optional] You may kindly keep all `weight_kg` values as floats.